examples/rn-mbnb/README.md

```diff
@@ -0,0 +1,143 @@
+│   ├── AnonAuth.tsx          - Anonymous authentication component
+│   ├── AppWrapper.ts         - Main wrapper with Realm Providers
+│   ├── localModels.ts     		- Local only realm model schema
+│   ├── localRealm.ts         - Local realm context and hooks
+│   ├── syncedModels.tsx      - Synced realm model schema
+│   └── syncedRealm.tsx       - Synced realm context and hooks
+├── App.js                      - Entry point
+├── sync.config.js                - Add App ID
+├── package.json                  - Dependencies
+└── README.md                     - Instructions and info
+```
+
+### Note: Using Multiple Realms at the Same Time
+
+This app uses multiple Realms, which have been configured using `createRealmContext`.  This creates separate providers and hooks to access either the local-only Realm or the synced Realm.
+
+#### Realm Configuration
+
+When [opening a Realm](https://www.mongodb.com/docs/realm/sdk/react-native/sync-data/configure-a-synced-realm/), we can specify the behavior in the Realm configuration when opening it for the first time (via `newRealmFileBehavior`) and for subsequent ones (via `existingRealmFileBehavior`). We can either:
+* `OpenRealmBehaviorType.OpenImmediately`
+  * Opens the Realm file immediately if it exists, otherwise it first creates a new empty Realm file then opens it.
+  * This lets users use the app with the existing data, while syncing any changes to the device in the background.
+* `OpenRealmBehaviorType.DownloadBeforeOpen`
+  * If there is data to be downloaded, this waits for the data to be fully synced before opening the Realm.
+
+This app opens a Realm via `RealmProvider` (see [AuthenticatedApp.tsx](./frontend/app/AuthenticatedApp.tsx)) and passes the configuration as props. We use `OpenImmediately` for new and existing Realm files in order to use the app while offline.
+
+> See [OpenRealmBehaviorConfiguration](https://www.mongodb.com/docs/realm-sdks/js/latest/types/OpenRealmBehaviorConfiguration.html) for possible configurations of new and existing Realm file behaviors.
+
+## Getting Started
+
+### Prerequisites
+
+* [Node.js](https://nodejs.org/)
+* [React Native development environment](https://reactnative.dev/docs/environment-setup?guide=native)
+  * Refer to the **"React Native CLI Quickstart"**.
+
+### Set up an Atlas Database with a Sample Dataset
+
+1. [Deploy a free Atlas cluster](https://www.mongodb.com/docs/atlas/getting-started/#get-started-with-atlas) and create an Atlas database.
+2. [Load the Sample Airbnb Dataset](https://www.mongodb.com/docs/atlas/sample-data/) into your Atlas database.
+    * Several databases and collections exist in the sample dataset, but we will only be using the `sample_airbnb` database and its `listingsAndReviews` collection.
+3. [Create a Search Index](https://www.mongodb.com/docs/atlas/atlas-search/tutorial/create-index/) with an Index Name of `airbnb`.  This will be used for Atlas Search within the application
```

Comment:
the search index name seems to be case sensitive and the backend is looking for `airBnB`, causing search to fail. So either change it here or in the backend (I'd lean towards backend as it's more likely people will end up typing all lowercase).

Reply:
good point.  I can update this.

Reply:
should also change it to `mbnb`